### PR TITLE
set up unit tests environment

### PR DESCRIPTION
**Pull request by MaciejBaj**
Write example unit test for peers upsert, mock modules with sinon, provide db access to test modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
     "jshint": "=2.9.3",
     "mocha": "=3.1.0",
     "moment": "=2.15.1",
+    "sinon": "^1.17.7",
```
Comment (karmacoma) [inline package.json]: Please fixate version like other dependenices. Not sure if `=` best approach, but at least provides some assurance of no unexpected surprises.

File: test/common/globalBefore.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+/**
+ * @param {string} table
+ * @param {Logger} logger
+ * @param {Object} db
+ * @param {Function} cb
+ */
+function clearDatabaseTable(db, logger, table, cb) {
```
Comment (karmacoma) [inline test/common/globalBefore.js]: Code style: Add spacing between function name and arguments.

File: test/common/initModule.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+			},
+			genesisblock: genesisblock
+		}));
+	};
+
+	/**
+	 * @param {Function} Module
+	 * @param {Function} cb
+	 */
+	this.initWithDb = function(Module, cb) {
```
Comment (karmacoma) [inline test/common/initModule.js]: Code style: Add spacing between function name and arguments.

File: package.json (hunk 1 above)
Comment (MaciejBaj) [inline package.json]: sure, thx

**Comment by MaciejBaj** [comment]
@karmacoma thx, review updated

**Pull request merged**